open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/manage/validator/VRPConsentRequestValidator.java

```diff
@@ -0,0 +1,608 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.openbanking.accelerator.consent.extensions.manage.validator;
+
+import com.wso2.openbanking.accelerator.common.util.ErrorConstants;
+import com.wso2.openbanking.accelerator.consent.extensions.common.ConsentExtensionConstants;
+import com.wso2.openbanking.accelerator.consent.extensions.util.ConsentManageUtil;
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Iterator;
+
+/**
+ * Consent Manage validator class for Variable Recurring Payment Request Validation.
+ */
+public class VRPConsentRequestValidator {
+
+    private static final Log log = LogFactory.getLog(VRPConsentRequestValidator.class);
+
+    /**
+     * Method to validate a variable recurring payment request.
+     * This method performs validation on the variable recurring payment request.
+     * It checks the validity of the  data body, the initiation payload, control parameters,
+     * and ensures that the risk information is present. If any validation fails, the method returns a detailed
+     * validation response indicating the error. If all validations pass, the returned validation response
+     * indicates that the initiation request is valid.
+     *
+     * @param request The initiation object containing the variable recurring payment initiation request.
+     * @return A validation response object indicating whether the initiation request is valid.
+     */
+    public static JSONObject validateVRPPayload(Object request) {
+
+        JSONObject validationResponse = new JSONObject();
+        validationResponse.put(ConsentExtensionConstants.IS_VALID, true);
```

Comment:
Shouldn't we keep the validationResponse = `false` in here?, If there's no issues then only we can make this true right?